test/e2e/daemon_set.go

```diff
@@ -61,8 +63,120 @@ var _ = Describe("Daemon set", func() {
 		f.afterEach()
 	})
 
-	It("should launch a daemon pod on every node of the cluster", func() {
-		testDaemonSets(f)
+	It("should run and stop simple daemon", func() {
+		ns := f.Namespace.Name
+		c := f.Client
```

Comment:
Sorry - perhaps my previous review comment was not clear.  ns and c need to be assigned in BeforeEach() and not here (and not in the other test either). Make sense?